src/python/pants/build_graph/app_base.py

```diff
@@ -212,8 +212,8 @@ def __init__(self,
                archive=None,
                **kwargs):
     """
-    :param string binary: Target spec of the ``jvm_binary`` that contains the
-      app main.
+    :param string binary: Target spec of the ``jvm_binary`` or the ``python-binary``
```

Comment:
probably `python_binary` instead.

Reply:
oof, thanks yi - this had slipped and i didn't get github notifications till now.